tests/integration/balance/BalanceTest.php

```diff
@@ -1,17 +1,16 @@
 <?php
 class BalanceTest extends BaseTest
 {
-    public function setUp()
+    protected function setUp(): void
     {
         parent::setUp();
         $this->client = new \MessageBird\Client('YOUR_ACCESS_KEY', $this->mockClient);
     }
 
-    /**
-     * @expectedException     MessageBird\Exceptions\ServerException
-     */
     public function testReadBalance()
     {
+        $this->expectException('MessageBird\Exceptions\ServerException');
+        $this->expectException(\MessageBird\Exceptions\ServerException::class);
```

Comment:
2?

Reply:
Ah yes, I did this one manually and the rest were actually automated via https://github.com/rectorphp/rector/blob/master/docs/AllRectorsOverview.md#exceptionannotationrector.

I'll try to change all of them to this format with some regexing.